addons/udes_priorities/models/priority_groups.py

```diff
@@ -0,0 +1,29 @@
+from odoo import models, fields, api, _
+
+
+class UdesPriorityGroup(models.Model):
+    _name = "udes_priorities.priority_group"
+    _order = "id"
+
+    name = fields.Char()
+    priority_ids = fields.Many2many(
+        "udes_priorities.priority", column1="group_id", column2="priority_id", string="Priorities"
+    )
+    active = fields.Boolean(default=True)
+    picking_type_ids = fields.Many2many(
+        "stock.picking.type",
+        string="Stock Transfer Types",
+        help="The stock picking types this priority is visible on",
+        compute="_compute_picking_types",
+    )
+
+    @api.multi
+    def _compute_picking_types(self):
+        for group in self:
+            group.picking_type_ids = group.mapped("priority_ids.picking_type_ids")
+
+    @api.depends("picking_type_ids.active")
+    def _deactivate_group(self):
```

Comment:
Why do we need this?

Reply:
So we don't present priority groups to the user with no priorities